### PR TITLE
added collapsible sidebar

### PR DESCRIPTION
**Pull request by LalitNM**
I added sidebar. @abhigyank have a look.

### CODE REVIEW

**Comment by abhigyank** [comment]
Awesome work! :rocket: I love it.

One minor thing that would make it perfect - when the side bar is open, if I click anywhere outside the sidebar, then also it should close. (Now it closes with only the button click). Is it possible to do this?

File: public/style.css [hunk 1]
```diff
@@ -96,19 +106,32 @@ body {
   -webkit-transform: translate(-50%, -50%);
   transform: translate(-50%, -50%);
 }
+
+.sidebar-btn {
```
Comment (abhigyank) [inline public/style.css:110]: Can you also add `font-size: 20px` here? That will make it a bit bigger.

**Comment by LalitNM** [comment]
> Awesome work!  I love it.
> 
> One minor thing that would make it perfect - when the side bar is open, if I click anywhere outside the sidebar, then also it should close. (Now it closes with only the button click). Is it possible to do this?

Let me try that.

File: public/style.css (hunk 1 above)
Comment (LalitNM) [reply public/style.css:110]: Will do this in next commit.

**Comment by LalitNM** [comment]
@abhigyank I changed `font-size` of `sidebar-btn`, but the functionality of 'closing sidebar on clicking anywhere outside sidebar' creates some additional bugs. So leave that for future developers :smile:.

By the way you can see those bugs after adding these lines in script.js:

function HideNav() {
  const navbar = document.querySelector(".left");
  if (navbar.style.left == '0px') {
    navbar.style.left = '-300px';
    right.style.width = "100%";
    var toggle = document.querySelector("#toggle-icon");
    toggle.className = '';
    toggle.className += 'fa fa-angle-right';
  }
}

$(document).on('click', function(event) {
  if (!$(event.target).closest('.left').length) {
    HideNav()
  }
});

**Comment by abhigyank** [comment]
@LalitNM  What bugs are you referring to that this code creates?

**Comment by LalitNM** [comment]
Umm... nothing big, the sidebar disappears when anyone creates new room. Because the person has to click on room after creating it to see conversation, I thought it would be good if sidebar remains visible even after creating a room.

**Comment by abhigyank** [comment]
@LalitNM Can you resolve the merge conflicts by rebasing to the latest master so that I can merge this PR?

**Comment by LalitNM** [comment]
Sure!

**Comment by LalitNM** [comment]
Done!

**Comment by abhigyank** [comment]
Merged :rocket:  Good work @LalitNM  !

**Pull request merged**